Taskfile.yaml

```
@@ -0,0 +1,51 @@
+version: "3"
+
+tasks:
+  clean:
+    cmds:
+      - go run ./internal/entclean/main.go --path=internal/ent {{ .CLI_ARGS }}
+      - "rm -rf internal/ent/generated/^doc.go"
+      - "rm -rf internal/api/^resolver.go"
+      - "rm -f schema/ent.graphql"
+      - rm -f schema.graphql
+      - rm -rf internal/testclient/
+  gen:
```

Can this be ent, since thats what is being generated ?

```suggestion
  ent:
```